src/pages/Host.tsx

```diff
@@ -79,7 +80,7 @@ export const HostCore: React.FC = () => {
     boolean
   >(false);
 
-  usePageTitle(`Host${hostUrl ? ` - ${hostUrl}` : ""}`);
+  usePageTitle(`Host${hostId ? ` - ${hostId}` : ""}`);
```

Comment:
should we show `host?.displayName` when available?

Reply:
I don't think it's necessary because it doesn't seem to be popular.